lib/fog_extensions/libvirt/server.rb

```diff
@@ -11,14 +11,6 @@ def to_s
         name
       end
 
-      def cpu_mode
-        attributes[:cpu][:mode]
-      end
-
-      def cpu_mode=(cpumode)
-        attributes[:cpu][:mode] = (cpumode == 'default') ? nil : cpumode
-      end
-
```

Comment:
It may be that this was also how it was previously set for the API, but I'm not sure it was ever exposed properly.